backend/src/main/java/com/summerHack/diningTogether/model/User.java

```diff
@@ -41,7 +41,9 @@ public class User {
 
     @OneToMany(fetch = FetchType.LAZY, mappedBy = "provider")
     private List<Food> foods;
-
+    @Column(name = "verification_code", length = 64)
+    private String verificationCode;
```

Comment:
I would rather put this into a separate table because it is just a temporary data, and we don't need to keep it all the time. And usually people put it into Redis instead of permanent database.

Reply:
So we can add time out to the verification code.